### DISCUSSION TITLE
Retiring a currently used sponsorship tier

### DISCUSSION THREAD

**Discussion by FerdinandCoumau** [community/community#40436] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I would like to retire a sponsorship tier that is currently used by a few sponsors.

Does this mean that the sponsorships will stop? Or it won't change anything for the current sponsors?

Thanks by advance for your help!

**Comment by dthoma1** [comment, marked as answer]
Existing sponsors will remain on the [retired tier](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/managing-your-sponsorship-tiers) until they change their sponsorship tier, cancel their sponsorship, or their one-time sponsorship period expires.
